brave-spark-framework-filter/src/main/java/com/github/kristofa/brave/spark/BraveSparkExceptionHandler.java

```diff
@@ -0,0 +1,57 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.ServerTracer;
+import com.github.kristofa.brave.http.HttpServerResponseAdapter;
+import spark.ExceptionHandler;
+import spark.Request;
+import spark.Response;
+import zipkin.Constants;
+
+import static com.github.kristofa.brave.internal.Util.checkNotNull;
+
+/**
+ * All your ExceptionHandler implements must be wrapped by BraveSparkExceptionHandler,
+ * If not,when exception happens,server span can't be generated,because AfterFilters won't be executed.

```

Comment:
watch for spacing. Also, wrap types in `{@link ExceptionHandler}` or `{@linkplain ExceptionHandler}`